src/Microsoft.SqlTools.ServiceLayer/SqlProjects/Contracts/DatabaseReferences/AddDatabaseReferenceParams.cs

```diff
@@ -18,8 +16,8 @@ public abstract class AddDatabaseReferenceParams : SqlProjectParams
         public bool SuppressMissingDependencies { get; set; }
 
         /// <summary>
-        /// SQLCMD variable name for specifying the other database this reference is to, if different from that of the current project
+        /// Database 
```

Comment:
Match the comment in DacFx for more detail on when this is used vs DatabaseVariable? `Literal name used to reference another database in the same server, if not using SQLCMD variables`